source/includes/indexes/indexes.py

```diff
@@ -88,26 +88,56 @@
 collection.create_search_index(index)
 # end-create-search-index
 
+# start-create-vector-search-index
+
+from pymongo.operations import SearchIndexModel
+
+search_index_model = SearchIndexModel(
+  definition={
+    "fields": [
+      {
+        "type": "vector",
+        "numDimensions": <numberOfDimensions>,
+        "path": "<fieldToIndex>",
+        "similarity": "euclidean | cosine | dotProduct"
+      }
+    ]
+  },
+  name="<nameOfIndex>",
+  type="vectorSearch",
+)
+
+collection.create_search_index(model=search_index_model)
+
+# end-create-vector-search-index
+
 # start-create-search-indexes
-index_one = {
-    "definition": {
-        "mappings": {
-            "dynamic": True
-        }
-    },
-    "name": "my_index",
-}
 
-index_two = {
-    "definition": {
+index_one = SearchIndexModel(
```

Comment:
S: rename var to be more descriptive
```suggestion
search_idx = SearchIndexModel(
```